viewing_party/party.py

```diff
@@ -1,23 +1,189 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
+#DONE
+import re
+
+
+def create_movie(movie_title, genre, rating):
+    dict = {}
+    
+    if movie_title is None:
+        return None
+    elif genre is None:
+        return None
+    elif rating is None:
+        return None
+    else:
+        dict["title"] = movie_title
+        dict["genre"] = genre
+        dict["rating"] = rating
+        return dict
+    pass
+
+#DONE
+def add_to_watched(user_data, movie):
+    emptyDict = {}
+    user_data["watched"].append(emptyDict)
+    user_data["watched"][0]["title"] = movie["title"]
+    user_data["watched"][0]["genre"] = movie["genre"]
+    user_data["watched"][0]["rating"] = movie["rating"]
+    return user_data
+    pass
+
+#DONE
+def add_to_watchlist(user_data, movie):
+    emptyDict = {}
+    user_data["watchlist"].append(emptyDict)
+    user_data["watchlist"][0]["title"] = movie["title"]
+    user_data["watchlist"][0]["genre"] = movie["genre"]
+    user_data["watchlist"][0]["rating"] = movie["rating"]
+    return user_data
+    pass
+
+#DONE
+def watch_movie(janes_data, movie_title):
+    for i in range(len(janes_data["watchlist"])):
+        if janes_data["watchlist"][i]["title"] == movie_title:
+            removed = janes_data["watchlist"].pop(i)
+            janes_data["watched"].append(removed)
+    return janes_data
     pass
 
+
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+#DONE
+def get_watched_avg_rating(janes_data):
+    sum = 0
+    if len(janes_data["watched"]) == 0:
+        return 0
+    for movie in janes_data["watched"]:
+        sum += movie["rating"]
+    return sum/(len(janes_data["watched"]))
+    pass
+
+#DONE
+def get_most_watched_genre(data):
+    genreDict = {}

```

Comment:
Python variables should use snake case as described in Python's style guide https://peps.python.org/pep-0008/#naming-conventions